commons/connector/fitbit/fitbit_skin_temperature.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitSkinTemperature",
+  "doc": "Skin temperature (tempSkin) data from fitbit device. tempSkin measures skin temperature data for a date range. It only returns a value for dates on which the Fitbit device was able to record Temperature (skin) data and the maximum date range cannot exceed 30 days. Temperature (Skin) data applies specifically to a user’s “main sleep,” which is the longest single period of time asleep on a given date.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)." },
+    { "name": "nightlyRelative", "type": "float", "doc": "The user's average temperature during a period of sleep. It is displayed to the user as a delta from their baseline temperature in degrees Celsius or Fahrenheit depending on the country specified in the Accept-Language header."},
```

Comment:
suggest to rename to `relativeTemperature`. In implementation, please use an `Accept-Language` that will result in Celcius. In the docs and in the data, use Celcius. Then you can remove the phrase
>  or Fahrenheit depending on the country specified in the Accept-Language header

Reply:
Thanks for the detail explanation.
Done.